### ISSUE TITLE
Contact Us or support option in settings

### ISSUE THREAD

**Issue by GianniCarlo** [GianniCarlo/Audiobook-Player#59]
Suggestion by @Daniel-Katz in #58 . We really need to make it easy for the users to report issues. One easy option would be to link this repo. The other option would be to let them send an email with their configured account in their iPhone. This is open to debate to see what would be the best approach

**Comment by pichfl** [comment]
I keep thinking about this and I'm not sure what would be the best way going forward with this. GitHub is not exactly a good way of handling support issues, but as this is an open source effort neither is having some sort of email account. I'll investigate platforms and services for this.

**Comment by pichfl** [comment]
After some research I found Zoho Desk, which has a [free plan](https://www.zoho.eu/desk/pricing.html) and [Front](https://frontapp.com), which is not free but I contacted them if they would be interested in providing a free account for our open source efforts. 

I'd wait until they respond. After that we can display a simple support email form or even just an email to write to and which we can access. I hope we won't be stuck doing this alone and someone else will be interested in joining this project as well.
